governance-charter.md

```diff
@@ -54,50 +52,8 @@ project groups like the [Technical Committee](./tech-committee-charter.md).
 
 ## Committee Structure
 
-### Establishment of a governance committee
-
-To bootstrap the process of OpenTelemetry governance, 5 individuals (Ben
-Sigelman, Bogdan Drutu, Sergey Kanzhelev, Sarah Novotny, and Yuri Shkuro) were
-identified to be the *Bootstrap Committee*, to provide the initial process that
-can bootstrap the remainder of the process.
-
-The Bootstrap Committee will be replaced by the elected OpenTelemetry Governance
-Committee. Ultimately, the **OpenTelemetry Governance Committee will consist
-of 9 individual members** of the community **elected for 2 year terms**. The
-terms will be **staggered with 1 year elections** (alternating 4 seats and 5
-seats).
-
-To provide a level of continuity as this process is established, the initial
-committee will include continuity members, expanding the size to 9 members.
-Note that from the 2020 election to the 2021 election, the committee will
-include 11 members.
-
-In 2019 as this approval is enacted, the committee consists of the
-members of the bootstrap committee (Ben Sigelman, Bogdan Drutu, Sarah Novotny,
-Sergey Kanzhelev, Yuri Shkuro) plus 4 positions to be elected from the
-community. Of the 4 members that are elected, 2 will have a two year term and 2
-will have one year term. 
-
-One year later (in 2020) there will be an election to fill the two seats
-opening up, each with a two year term. The continuity members of the original
-bootstrap committee will continue to serve.
-
-One year after that (2021), there will be an election to fill the 5 open
-seats on the governance committee. The previous continuity members are
-eligible to run for these seats.
-
-The committee will continue to iterate with alternating elections of four and
-five members each year. 
-
-For clarity, a table describing this process is given below:
-
-| *Year* | *Continuity* | *Election Cohort #1*    | *Election Cohort #2*    | GC size post-election |
-| ------ | ------------ | ----------------------- | ----------------------- | --------------------- |
-| 2019   | 5 people     | **2 people** (2yr term) | **2 people** (1yr term) | 9 people (+4)         |
-| 2020   | -            | -                       | **4 people** (2yr term) | 11 people (-2, +4)    |
-| 2021   | 0 people     | **5 people** (2yr term) | -                       | 9 people (-7, +5)     |
-| 2022   | -            | -                       | **4 people** (2yr term) | 9 people (-4, +4)     |
-| 2023   | -            | **5 people** (2yr term) | -                       | 9 people (-5, +5)     |
+The **OpenTelemetry Governance Committee consists of 9 individual members** of the community **elected for 2 year terms**. The
+terms are **staggered with 1 year elections** (alternating 4 seats and 5 seats).
```

Comment:
```suggestion
terms are **staggered with 1 year elections** (alternating 4 seats in even years and 5 seats in odd years).
```

Reply:
Based on the fact that in the case of resignation/loss of a GC member a special election is hold and the elected GC member will serve out the remainder of the term for the person they are replacing, this appears to be a safe change as well. Thanks!

(I changed the formatting, that's what outdated the comment)